packages/fast-tooling-react/src/navigation/navigation.style.ts

```diff
@@ -0,0 +1,75 @@
+import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
+import { applyTriggerStyle, insetStrongBoxShadow } from "./navigation.utilities.style";
+import { accent, background300, background800, foreground300 } from "../style/constants";
+
+export interface NavigationClassNameContract {
+    navigation?: string;
+    navigation_item?: string;
+    navigation_itemLink?: string;
+    navigation_itemLink__active?: string;
+    navigation_itemExpandListTrigger?: string;
+    navigation_itemExpandListTrigger__active?: string;
+    navigation_itemList?: string;
+}
+
+const styles: ComponentStyles<NavigationClassNameContract, {}> = {
+    navigation: {
+        fontSize: "12px",
```

Comment:
should we use our typeramp?

Reply:
I think this will create a dependency on the MSFT package and I'm not sure we want to do that.

Reply:
@chrisdholt is correct, the tooling are agnostic to msft styles.